rest-openehr/src/main/java/org/ehrbase/rest/openehr/OpenehrDefinitionQueryController.java

```diff
@@ -107,11 +115,10 @@ public ResponseEntity<QueryDefinitionResponseData> getStoredQueryVersion(
     }
 
     @EhrbaseAuthorization(permission = EhrbasePermission.EHRBASE_QUERY_CREATE)
-    @RequestMapping(
-            value = {"/{qualified_query_name}/{version}", "/{qualified_query_name}"},
-            method = RequestMethod.PUT)
+    @PutMapping(value = {"/{qualified_query_name}/{version}", "/{qualified_query_name}"})
```

Comment:
Missing: `produces = {TEXT_PLAIN_VALUE, APPLICATION_JSON_VALUE}`, as it deviates from the defaults

Reply:
done